cmd/admin-user-add-policy.go

```diff
@@ -68,7 +80,11 @@ func mainAdminUserPolicy(ctx *cli.Context) error {
 	client, err := newAdminClient(aliasedURL)
 	fatalIf(err, "Cannot get a configured admin connection.")
 
-	fatalIf(probe.NewError(client.SetUserPolicy(args.Get(1), args.Get(2))).Trace(args...), "Cannot set user policy for user")
+	if ctx.Bool("sts-kerberos") {
+		fatalIf(probe.NewError(client.SetKrbUserPolicy(args.Get(1), args.Get(2))).Trace(args...), "Cannot set user policy for user")
```

Comment:
don't see this function SetKrbUserPolicy defined in this or minio-go repo. Your kerberos.go example in minio/minio could also include setting Kerberos policy prior to ListBuckets() so that the flow is clear to user.

Reply:
This function is in the minio/minio's madmin package and is part of the server PR. I will update kerberos.go.

To get this PR working we need to first merge minio-go PR, then minio server PR.